src/Symfony/Component/Form/ChoiceList/Factory/ChoiceListFactoryInterface.php

```diff
@@ -76,12 +76,13 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, callable $va
      * match the keys of the choices. The values should be arrays of HTML
      * attributes that should be added to the respective choice.
      *
-     * @param array|callable|null $preferredChoices The preferred choices
-     * @param callable|false|null $label            The callable generating the choice labels;
-     *                                              pass false to discard the label
-     * @param array|callable|null $attr             The callable generating the HTML attributes
+     * @param array|callable|null $preferredChoices           The preferred choices
+     * @param callable|false|null $label                      The callable generating the choice labels;
+     *                                                        pass false to discard the label
+     * @param array|callable|null $attr                       The callable generating the HTML attributes
+     * @param array|callable      $labelTranslationParameters The parameters used to translate the choice labels
      *
      * @return ChoiceListView The choice list view
      */
-    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, callable $index = null, callable $groupBy = null, $attr = null);
+    public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, callable $index = null, callable $groupBy = null, $attr = null/*, $labelTranslationParameters = []*/);
```

Comment:
We should trigger a deprecation for implementations of this interface that do not have the new argument.

Reply:
@VincentLanglet It looks like you forgot to implement this feedback.

Reply:
@fabpot I added deprecation for all the three `createView` now. I hope it was the right way to do it.